snowfall/models/interface.py

```diff
@@ -0,0 +1,20 @@
+from torch import nn
+
+
+class AcousticModel(nn.Module):
+    """
+    AcousticModel specifies the common attributes/methods that
```

Comment:
I am afraid these common attributes will not be saved `by save_checkpoint` by default.

Reply:
I added handling them to save/load checkpoint; I think we should make these a part of the model class actually, I can do it in a future PR.